academy/1-what-is-cosmos/4-atom-staking.md

```diff
@@ -55,52 +63,69 @@ Keplr is a non-custodial wallet.
+
+When you have recorded the phrase securely, click "Next" to continue.
+
+On the next screen, you are first prompted to provide two words from your mnemonic seed phrase.
+
+![Keplr mnemonic verify your recovery page, part 1](/academy/1-what-is-cosmos/images/keplr_new_step_07.png)
+
+You must then give your wallet a name, and assign it a strong password. 
+
+![Keplr mnemonic verify your recovery page, part 2](/academy/1-what-is-cosmos/images/keplr_new_step_08.png)
+
 * Keplr asks you for the **encryption password**, which is used to encrypt your mnemonic on your computer. For the password, keep in mind:
     * It should be a strong password.
     * You should also save it securely.
     * Keplr will ask you for it every time you open the wallet and for important actions.
 * If your computer or the encrypted mnemonic seed file is stolen, your wallet assets are only as protected as your password is strong.
 
-![Keplr mnemonic seed and creating a password page](/academy/1-what-is-cosmos/images/keplr-mnemonic-step.png)
-
-When you are ready, click _Next_.
-
-If you saved your seed, you should be able to complete the next step: click on each word in the correct order.
+Click "Next" to continue.
 
-![Keplr mnemonic recall prompt: Selecting the correct words of your mnemonic](/academy/1-what-is-cosmos/images/keplr-mnemonic-recall-prompt.png)
+On the next screen, select the chains you want to link to, then click "Save" to continue.
 
-When done, the _Register_ button should become active:
+![Keplr select chains page](/academy/1-what-is-cosmos/images/keplr_new_step_09.png)
 
-![Keplr mnemonic recall done](/academy/1-what-is-cosmos/images/keplr-mnemonic-recall-done.png)
+Your account is now created. Click "Keplr dashboard" to open the [Keplr dashboard page](https://wallet.keplr.app/#/dashboard) where you can review your wallet.
 
-Click _Register_. Your Keplr wallet is now fully created:
+![Keplr all set](/academy/1-what-is-cosmos/images/keplr_new_step_10.png)
 
-![Keplr all set](/academy/1-what-is-cosmos/images/keplr-all-set.png)
+This is your Keplr dashboard. It shows your first public address in the form `cosmos1...`, displays that you hold zero ATOM, and you can also see how many assets from other supported networks you have. Be aware that initially you may have a number of messages in your Inbox which request approval to complete registration.
 
-Click the extension icon again. It shows your first public address in the form `cosmos1...`, and that you hold zero ATOM.
+![Keplr dashboard](/academy/1-what-is-cosmos/images/keplr_new_step_11.png)
 
-Click the drop-down menu labeled _Cosmos_ at the top of the screen to see how many assets from other supported networks you have:
+At the top of the dashboard you can click to copy the address of your wallet. This will be useful when you move on to purchase some ATOM from your preferred exchange (the following example will use the Kraken exchange).
```

Comment:
```suggestion
At the top of the dashboard you can click to copy the address of your wallet. This will be useful when you move on to purchase some ATOM from your preferred exchange.
```

I would not mention the one used.